docs/apps/ansys.md

```diff
@@ -0,0 +1,44 @@
+## ANSYS
+ANSYS offers a comprehensive software suite that spans the entire range of physics, providing access to virtually any field of engineering simulation that a design process requires ([ANSYS - Products](https://www.ansys.com/products)). *ANSYS Academic engineering simulation software*  (hereinafter **AAESS**) is used by thousands of universities globally for undergraduate students to learn physics principles, for researchers to solve complex engineering problems and for postgraduate students to produce data for their master’s theses or doctoral dissertations ([ANSYS - Academic](https://www.ansys.com/academic)).
+
+## License
+
+AAESS products are proprietary software. CSC - IT Center for Science Ltd. has these licences available on CSC's server platforms. The licenses are only for academic use.  See about use limitations from link given below (Ansys Academic - Terms and Conditions), and there, LICENSING AND TERM OF USE.
+
+## Available
+
+CSC's AAESS product licenses are available on CSC's computing platforms [Taito and Puhti](https://research.csc.fi/csc-s-servers).  Latest AAESS products will be available on the servers, and earlier versions installation is also possible.  All installed versions are maintained on the servers.
+
+Notice that Taito server will be decommisioned in the end of December 2019.
+
+## Usage
+
+After login on the server, give command
+
+    module available
+
+and check lines *ansys/(version number)*. For example to load Ansys version 19.2., give command
+
+    module load ansys/19.2
+
+There are examples of **batch job files** available on Puhti server:
+
+    Ansys CFX:        /appl/soft/eng/ansys_inc/example_batch_job_files/parjob_puhti_cfx
+
+    Ansys Fluent:     /appl/soft/eng/ansys_inc/example_batch_job_files/parjob_puhti_fluent
+
+    Ansys Structural: /appl/soft/eng/ansys_inc/example_batch_job_files/parjob_puhti_struct
```

Comment:
These batch job script do not conform to the style guidelines:
- Short flags should be changed to long flags
- There is no point in setting `--ntasks-per-node=4` after we already set `-n` and `-N` (it will have no effect.)

Reply:
What is short/long flag?
ntasks-per-node is in purpose there.  Indeed, it is not needed in the test case, but important flag (flag ;-) when fitting some maybe bigger job in.

Reply:
Moi,

[asillanp@puhti-login1 ~]$ more /appl/soft/eng/ansys_inc/example_batch_job_files/parjob_puhti_struct
#!/bin/bash -l
#SBATCH -J structTest
#SBATCH -e %j.err
#SBATCH -o %j.out
#SBATCH -N 1
#SBATCH -n 4
#SBATCH --ntasks-per-node=4
#SBATCH --time=00:10:00
#SBATCH --partition=test
#SBATCH --account=project_XXXXXX

ainakin tuossa on puolet lyhyitä ja puolet pitkiä vipuja. Tarkoitus on käyttää pelkästään pitkiä. Eikö tuo lisäksi ole outoa pyytää sekä ntasks että ntasks-per-node? En keksi muuta syytä tuohon kuin jos haluat käyttää vain osan useammasta solmusta (tuskin järkevää?). Eikö paras tapa olisi --nodes=X ja --ntasks-per-node=40, isoihin töihin ja --nodes=1 ja --ntasks-per-node=<1-40> pienempiin?

Projekti piti kans olla muodossa <project> (kaikkien nimi ei ole tuota muotoa mikä on yllä)

Terveisin,
Atte

Reply:
OK, now I understood - flags in the batch job files. => corrected.